client/src/components/RelatedProducts/Card.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+
+class Card extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+  render() {
+    return (
+      <div>Category: {this.props.productInfo.category}
+        <div className='card-itself'>
+          <img
+            className='default-image'
+            src={this.props.productInfo.url}
+            alt={this.props.productInfo.name}
+          />
+        </div>
+        <div>Name: {this.props.productInfo.name}</div>
+        <div>Price: {this.props.productInfo.default_price}</div>
+        <div>Rating: ★★★☆☆</div>
```

Comment:
I'm very interested in the rendering of the stars and what you are using to do this.  Perhaps at standup we can have a small discussion about it.  Is this currently just a place holder star rating, or are you formatting that data somehow? 
How are you using the font awesome import in this file?